src/Symfony/Component/Messenger/TraceableMessageBus.php

```diff
@@ -16,6 +16,8 @@
  */
 class TraceableMessageBus implements MessageBusInterface
 {
+    private static $calls = 0;
```

Comment:
This might not be ideal. Use a microtime to keep calls order instead?

Reply:
What's wrong with keeping PHP incrementing the keys with `$this->dispatchedMessages[]`?

Reply:
There are multiple buses, so multiple traceable buses. This ensures the calls order is kept, while iterating over each traceable buses in the datacollector to get their dispatched messages separately won't 😉

Reply:
That's what I suggested in https://github.com/symfony/symfony/pull/27202#discussion_r187465603 :p
I take that as a "Let's do it" then 😄 Thanks

Reply:
🙊